libvast/src/detail/process.cpp

```diff
@@ -63,4 +130,15 @@ caf::expected<path> objectpath(const void* addr) {
   return objectpath_static();
 }
 
+caf::settings get_status() {
```

Comment:
Since the output here differs depending on the platform, can you add an integration test that uses `jq '.path.to.rusage | keys'` to test whether all keys are available on all platforms?

Reply:
This is not possible, for example macOS does not expose information about swap.